ciao/main.go

```diff
@@ -0,0 +1,23 @@
+// Copyright © 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/ciao-project/ciao/ciao/cmd"
+)
+
+func main() {
+	cmd.Execute()
```

Comment:
Help seems to be missing for some of the commands:

```
Available Commands:
  add         
  attach      
  create      Create cluster objects.
  delete      
  detach      Detach various ciao objects
  evacuate    
  help        Help about any command
  list        List information about various ciao objects
  remove      
  restart     
  restore     Restore a node
  show        Show detailed information about a ciao object
  stop        
  update      Update various ciao objects

```

Reply:
Fixed this by rolling out more Short: fields.